src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -813,4 +726,22 @@ private boolean isUntrimmed(String value) {
     public static boolean isValidEmailAddress(String email) {
         return StringHelper.isMatching(email, REGEX_EMAIL);
     }
+
+    /////////////////////////////////////
+    // HELPER METHODS FOR INTERNAL USE //////////////////////////
+    // DUPLICATED IN FieldValidatorTest FOR EXTERNAL TESTS USE //
+    /////////////////////////////////////////////////////////////
```

Comment:
@wkurniawan07 I think the methods should not be public because other (non-test) classes should not be constructing error messages themselves, but should rather use the specific `getInvalidityInfoFor...()` provided.

I'll remove the comments and the duplicated methods in `FieldValidatorTest`